src/Surfnet/DataFixtures/AppFixtures.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Surfnet\DataFixtures;
+
+use Doctrine\Bundle\FixturesBundle\Fixture;
+use Doctrine\Common\Persistence\ObjectManager;
```

Comment:
Using the ObjectManager from `Doctrine\Common\Persistence\` is incompatible with super